applications/CoSimulationApplication/python_scripts/data_transfer_operators/sum_distributed_to_distributed.py

```diff
@@ -0,0 +1,43 @@
+# Importing the base class
+from KratosMultiphysics.CoSimulationApplication.base_classes.co_simulation_data_transfer_operator import CoSimulationDataTransferOperator
+
+# Other imports
+import numpy as np
+
+def Create(settings):
+    return SumDistributedToDistributed(settings)
+
+class SumDistributedToDistributed(CoSimulationDataTransferOperator):
```

Comment:
```suggestion
class SumMutlipleToMultiple(CoSimulationDataTransferOperator):
```

I also want to rename the other classes, since `distributed` is associated with MPI in Kratos